Remove 'Reflecting ARIA attributes' from experimental FX relnotes; add ARIA reflection to FX119 relnotes (#29521)
mdn/content: files/en-us/mozilla/firefox/releases/119/index.md

#### DOM

> - [ARIA](/en-US/docs/Web/Accessibility/ARIA) reflection is enabled for attributes that do not reference other elements; only non-IDREF attributes are reflected. This allows developers to get and set ARIA attributes on DOM elements directly via JavaScript APIs, rather than by using `setAttribute` and `getAttribute`. For example, `el.ariaPressed = "true";` is now supported in addition to `el.setAttribute("aria-pressed", "true");`. (See [Firefox bug 1785412](https://bugzil.la/1785412) for more details.)

Reviewer: - Should we say "now supported by default" because it could be enabled earlier but after setting the preference?
- Looking at a [previous release note](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/117) to keep the styling consistent for referencing the Fx bug.
- The addition of a quick example is really good here! 👍. I am wondering if `el` should be `<element>`. Or perhaps, we should provide an actual example like `button.ariaPressed = "true";`?

```suggestion
- [ARIA](/en-US/docs/Web/Accessibility/ARIA) reflection is now supported by default for attributes that do not reference other elements; only non-IDREF attributes are reflected. You can now get and set ARIA attributes on DOM elements directly via JavaScript APIs, rather than by using `setAttribute` and `getAttribute`. For example, `el.ariaPressed = "true";` is now supported in addition to `el.setAttribute("aria-pressed", "true");` ([Firefox bug 1785412](https://bugzil.la/1785412)).
```

Author: > * Should we say "now supported by default" because it could be enabled earlier but after setting the preference?

Yeah, I like that!

> * Looking at a [previous release note](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/117) to keep the styling consistent for referencing the Fx bug.

Thanks :)

> * The addition of a quick example is really good here! 👍. I am wondering if `el` should be `<element>`. Or perhaps, we should provide an actual example like `button.ariaPressed = "true";`?

Ooh, okay, I have changed it to `buttonElement.ariaPressed = "true";`, which I think is better and more clear. WDYT?
